graphql/schema/types/filters.graphql

```diff
@@ -12,11 +12,19 @@ input FindFilterType {
 }
 
 enum ResolutionEnum {
+  "144p", VERY_LOW
   "240p", LOW
+  "360p", R360P
```

Comment:
Couldn't think of a decent name for 360p. It was a standard resolution but never really had a label outside of "360p".